docs/configuration.asciidoc

```diff
@@ -0,0 +1,265 @@
+[[configuration]]
+== Configuration
+
+[float]
+=== Gradle configuration
+
+Configure your application at compile time within your application's `build.gradle` file:
+
+[source,groovy]
+----
+// Android app's build.gradle file
+plugins {
+    //...
+    id "co.elastic.apm.android" version "[latest_version]"
+}
+
+elasticApm {
+    // Minimal configuration
+    serverUrl = "https://your.elastic.server"
```

Comment:
We explain the default for every config other than this one. What is the default value?